samples/attested_tls/server/enc/server.cpp

```diff
@@ -151,20 +156,19 @@ int handle_communication_until_done(
             errbuf);
         goto done;
     }
-    printf(
```

Comment:
Have removed this as this makes the logs chatty in case of frequent health checks.